lti_tool/views.py

```diff
@@ -30,6 +30,16 @@ def jwks(request):
 class OIDCLoginInitView(View):
     """Handles OIDC 3rd-party login initiation for an LTI launch."""
 
+    """
+    Message strings used by `DjangoOIDCLogin.enable_check_cookies()`.
+    """
+    main_msg: str = (
+        "Your browser prevents embedded content from using cookies.  To work "
+        "around this, the content must be opened in a new tab or window.  "
+    )
+    click_msg: str = "Open a new tab or window now."
+    loading_msg: str = "Loading..."
```

Comment:
I'm not a Python typing expert, but I'm wondering if explicit type hints are necessary here? I would think that maybe `mypy` and other type checkers could infer that these are strings based on the assignment.

Reply:
That's true.  When I changed them from optional, I could've removed the rest of the typing.

Reply:
Removed.